coretex/cli/utils.py

```diff
@@ -14,3 +34,56 @@ def arrowPrompt(choices: List[Any]) -> Any:
     ])
 
     return answers["option"]
+
+
+def checkIfExpired(expirationDate: datetime) -> bool:
+    currentTime = datetime.utcnow().replace(tzinfo=timezone.utc)
+    if currentTime >= expirationDate:
+        return True
+    return False
+
+
+def refresh() -> None:
+    config = loadConfig()
+    apiExpirationDate = decodeDate(config["tokenExpirationDate"])
+    refreshExpirationDate = decodeDate(config["refreshTokenExpirationDate"])
+    isApiExpired = checkIfExpired(apiExpirationDate)
+
+    if not isApiExpired:
+        return None
+
+    isRefreshExpired = checkIfExpired(refreshExpirationDate)
+    if not isRefreshExpired:
+        refreshToken = config["refreshToken"]
+        response = networkManager.authenticateWithRefreshToken(refreshToken)
+    else:
+        username = config["username"]
+        password = config["password"]
+
+        response = networkManager.authenticate(username, password, False)
+        if response.hasFailed():
+            # check for codes user/server failure
+            raise RuntimeError("Something went wrong. Try configuring user again...")
+
+    jsonResponse = response.getJson(dict)
+    config["token"] = jsonResponse["token"]
+    config["tokenExpirationDate"] = jsonResponse["expires_on"]
+    saveConfig(config)
+
+
+def validate(exclude_options: Optional[List[str]]) -> Any:
+    if exclude_options is None:
+        exclude_options = []
+
+    def decorator(f: Any) -> Any:
```

Comment:
`f` should not be of type `Any`, type the function properly, as well as the return value.